libkbfs/folder_block_manager.go

```diff
@@ -1255,3 +1305,120 @@ func (fbm *folderBlockManager) clearLastQRData() {
 	fbm.wasLastQRComplete = false
 	fbm.lastReclamationTime = time.Time{}
 }
+
+func (fbm *folderBlockManager) doCleanSyncCache() (err error) {
+	defer fbm.cleanSyncCacheGroup.Done()
+	if !fbm.config.IsSyncedTlf(fbm.id) {
+		return nil
+	}
+	dbc := fbm.config.DiskBlockCache()
+	if dbc == nil {
+		return nil
+	}
+
+	ctx, cancel := context.WithCancel(fbm.ctxWithFBMID(context.Background()))
+	fbm.setCleanSyncCacheCancel(cancel)
+	defer fbm.cancelCleanSyncCache()
+
+	lState := makeFBOLockState()
+	recentRev := fbm.helper.getLatestMergedRevision(lState)
+
+	lastRev, err := dbc.GetLastUnrefRev(ctx, fbm.id)
+	if err != nil {
+		return err
+	}
+
+	if lastRev < kbfsmd.RevisionInitial {
+		if recentRev > kbfsmd.RevisionInitial {
+			// This can happen if the sync cache was created and
+			// populated before we started keeping track of the last
+			// unref'd revision. In that case, we just let the blocks
+			// from the old revision stay in the cache until they are
+			// manually cleaned up.
```

Comment:
What does "manually cleaned up" mean here?

Reply:
I was thinking of advising anyone we know that's syncing TLFs right now to just blow away their sync cache when we release this feature, and let everything re-sync.

Reply:
OK yeah let's do that.